src/guides/v2.3/rest/tutorials/bundle-product/create-bundle-products.md

```diff
@@ -0,0 +1,399 @@
+---
```

Comment:
The word "Cardio" has been misspelt around 40 times across the files, it appears as "Cardo" - please correct all of these.

Reply:
@BarnyShergold as per suggestion i have updated